src/app/TelemetryApp.svelte

```diff
@@ -24,29 +24,27 @@ function updateQueryString() {
   }
 }
 
-$: if (visible) {
+$: if ($store.token && visible) {
   updateQueryString($currentQuery);
 }
 
 onMount(() => { visible = true; });
-
 </script>
 
-<style>
-
-</style>
-<App>
-    <!-- <TelemetryAppBar /> -->
-    <!-- <TelemetryControls /> -->
-
-    <Main>
-        <MainActionBar>
-                <TelemetryAppBar />
-                <Search />
-                <TelemetryMainFilters />
-        </MainActionBar>
-        <Content>
-          <GraphicBody />
-        </Content>
-    </Main>
-</App>
+{#if $store.token}
```

Comment:
is the idea here that if `$store.token` doesn't exist, we won't see anything at all?

Reply:
Yep, exactly. I'll add a comment. :smiley: